### PR TITLE
Rename school service attributes

### PR DESCRIPTION
**Pull request by ollietreend**
## Changes proposed in this pull request

This PR renames the `claims` and `placements` boolean attributes to `claims_service` and `placements_service`.

This frees up the methods `School#claims` and `School#placements` so they can be used by more meaningful things – for example, to define a School `has_many` Claim association.

## Guidance to review

This is mostly just a 'find & replace' exercise, so there's not too much to see here.

I have made one functional change, though. I've removed the `claims` and `placements` scope from the top-level `School` model. I'd be open to opinions about whether this is a good change or not. I did it mainly to discourage use of the top-level `School` model, since each service has a namespaced `School` model that is already scoped.

## Link to Trello card

https://trello.com/c/2Uw4TwWm/141-rename-claims-and-placements-attributes-on-the-school-model

### CODE REVIEW

**Review by CatalinVoineag — APPROVED** [review]
Looks good!

File: app/forms/school_onboarding_form.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def school_exists?
   end
 
   def school_already_onboarded?
-    if school&.try(service.to_s)
+    if school&.try("#{service}_service?".to_s)
```
Comment (JamieCleare2525) [inline app/forms/school_onboarding_form.rb:28]: not a blocker but you probably don't need the `to_s`

**Review by JamieCleare2525 — APPROVED** [review]

**Review by elceebee — COMMENTED** [review]
Nice one -- good to spot this now as a problem rather than later when things are too complicated to unpick. 

For consistency, should we also change the bool on providers to follow the same pattern? I know it is just placements, not claims, but it feels right to make it match 
image

Also, apologies for being pedantic, but it would be good to keep the data-model.doc up-to-date with these changes.

**Review by elceebee — APPROVED** [review]

File: app/forms/school_onboarding_form.rb (hunk 1 above)
Comment (ollietreend) [reply app/forms/school_onboarding_form.rb]: Done! Thanks for pointing that out.

**Pull request merged**